### PR TITLE
feat: troubleshooting for dirty database

### PR DESCRIPTION
**Pull request by nityanandagohain**
Fixes https://github.com/SigNoz/engineering-pod/issues/1560

### CODE REVIEW

**Comment by nityanandagohain** [comment]
This doc will solve the issue where current users are facing problems. Once we have changes in otel collector https://github.com/SigNoz/signoz-otel-collector/pull/346 merged.

Will create another doc on how to migrate from older version to the version where the above is merged. @srikanthccv

**Review by srikanthccv — APPROVED** [review]

File: data/docs/userguide/logs_troubleshooting.mdx [hunk 1]
```diff
@@ -7,6 +7,96 @@ description: Instructions that should resolve most issues with logs
...
+```
+Similar error messages include
+```
+Dirty database version 11. Fix and force version.
+```
+```
+Dirty database version 12. Fix and force version.
+```
+
+To fix this issue, first disable to schema migrator.
```
Comment (ankit01-oss) [inline data/docs/userguide/logs_troubleshooting.mdx]:
```suggestion
To fix this issue, first disable the schema migrator.
```

File: data/docs/userguide/logs_troubleshooting.mdx [hunk 2]
```diff
@@ -7,6 +7,96 @@ description: Instructions that should resolve most issues with logs
...
+* On k8s you can delete the job.
+* On Docker you can delete the schema migrator container.
+
+#### Run The Migrations Manually.
```
Comment (ankit01-oss) [inline data/docs/userguide/logs_troubleshooting.mdx]:
```suggestion
#### Run The Migrations Manually
```

File: data/docs/userguide/logs_troubleshooting.mdx [hunk 3]
```diff
@@ -7,6 +7,96 @@ description: Instructions that should resolve most issues with logs
...
+        ```sql
+        KILL MUTATION where mutation_id='<mutation_id>'
+        ```
+    * If you have any doubt's please reach out to us on our community [slack](https://signoz.io/slack)
+* Run the migrations manually by running each command from the below files on the clickhouse console. If the 10th migration failed then start with 10th, else start with the version where it failed.
+    * 10:- https://github.com/SigNoz/signoz-otel-collector/blob/main/migrationmanager/migrators/logs/migrations/000010_body_ngram.up.sql
+    * 11:- https://github.com/SigNoz/signoz-otel-collector/blob/main/migrationmanager/migrators/logs/migrations/000011_add_instrumentation_scope.up.sql
+    * 12:- https://github.com/SigNoz/signoz-otel-collector/blob/main/migrationmanager/migrators/logs/migrations/000012_rename_instrumentation_scope.up.sql
+    * 13:- https://github.com/SigNoz/signoz-otel-collector/blob/main/migrationmanager/migrators/logs/migrations/000013_rename_instrumentation_scope.up.sql
+    While running them replace `{{.SIGNOZ_CLUSTER}}` with cluster.
```
Comment (ankit01-oss) [inline data/docs/userguide/logs_troubleshooting.mdx:48]:
```suggestion
    <br></br>
    While running them replace `{{.SIGNOZ_CLUSTER}}` with cluster.
```

File: data/docs/userguide/logs_troubleshooting.mdx [hunk 4]
```diff
@@ -7,6 +7,96 @@ description: Instructions that should resolve most issues with logs
...
+        The following should be present in the above response
+        ```sql
+        `tagType` Enum8('tag' = 1, 'resource' = 2, 'scope' = 3) CODEC(ZSTD(1)),
+        ```
+
+Once verified run the schema migrator again
+* On k8s run helm upgrade.
+* On Docker rerun the docker compose file.
+
+If you face any issue please reach out to us on [slack](https://signoz.io/slack)
```
Comment (ankit01-oss) [inline data/docs/userguide/logs_troubleshooting.mdx]:
```suggestion
If you face any issue please reach out to us on [slack](https://signoz.io/slack).
```

**Review by ankit01-oss — APPROVED** [review]

**Pull request merged**